docs/DafnyRef/Statements.md

```diff
@@ -649,6 +649,8 @@ function usesTuple() : int
 }
 ```
 
+The special assignment operator `:-` enables to quickly return if the value evaluates to an error-compatible type, see [Section 19.7](sec-update-failure).
```

Comment:
Is 'special assignment operator' the right name? What about 'assignment with failure operator' ?

I think there's a grammatical error in the short summary: "enables to quickly return" -> "enables quickly returning"

Consider rephrasing to "The assignment with failure operator `:-` returns [from the method] if the value evaluates to a failure value of a failure-compatible type, see [Section 19.7](sec-update-failure)".